src/components/Navigation/Navigation.stories.ts

```diff
@@ -0,0 +1,59 @@
+// Button.stories.ts
+
+import type { Meta, StoryObj } from "@storybook/react";
+import {
+  reactRouterParameters,
+  withRouter,
+} from "storybook-addon-remix-react-router";
+import Navigation from "./Navigation.tsx";
+
+const meta: Meta<typeof Navigation> = {
+  title: "Example/Navigation",
+  component: Navigation,
+  decorators: [withRouter],
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Navigation>;
+
+export const Home: Story = {
+  parameters: {
+    reactRouter: reactRouterParameters({
+      routing: { path: "/" },
+    }),
+  },
+};
```

Comment:
You should then be able to add a docs key to your `Home` object.

```js
 docs: {
    description: {
      story: "The Navigation with Home highlighted",
    },
  },
  ```

Reply:
Once you have added this to each it will give you a clear summary of each state of the component when you run story book